### PR TITLE
Jameson/add style prop to loader

### PR DESCRIPTION
**Pull request by Jameson13B**
This MR add the `style` prop to the Loader component to override styles.

### CODE REVIEW

File: src/components/Loader.js [hunk 1]
```diff
@@ -53,7 +55,7 @@ class Loader extends React.Component {
   styles = (theme) => {
     const color = this.props.color || theme.Colors.PRIMARY;
 
-    return {
+    return _merge({}, {
```
Comment (phantomxc) [inline src/components/Loader.js]: You can wrap this in an if checking for the prop so we aren't merging nothing if we aren't passing in any styles.

Comment (Jameson13B) [reply src/components/Loader.js]: @phantomxc Great idea. I have that updated and found a typo. Should be good to go. Thanks!

File: src/components/Loader.js [hunk 2]
```diff
@@ -53,47 +55,90 @@ class Loader extends React.Component {
...
+          borderTop: '3px solid ' + color,
+          borderRight: '3px solid transparent',
+          borderBottom: '3px solid transparent',
+          borderLeft: '3px solid transparent'
+        },
+        text: {
+          marginTop: '10px',
+          fontSize: '10px'
+        }
+      }, this.props.styles)
```
Comment (phantomxc) [inline src/components/Loader.js]: Lets move the styles to a const so it's not duplicated in your ternary.

**Comment by phantomxc** [comment]
:shipit:

**Pull request merged**